.github/shared/src/swagger.js

```diff
@@ -38,15 +38,26 @@ export class Swagger {
    * @returns {Promise<Set<Swagger>>}
    */
   async getRefs() {
+    await this.populateRefPaths();
```

Comment:
This can be simplified by adding an intermediate private function `async #getRefs()` that handles the lazy-load and always returns non-null.  Like we do here:

https://github.com/Azure/azure-rest-api-specs/blob/0752d8ee3182a9b35ae1fa6a4b94962b7642b087/.github/shared/src/readme.js#L186-L195

Reply:
Ok I see the pattern yeah this will look nice. Update incoming.